pkgs/tools/misc/mlocate/default.nix

```diff
@@ -1,8 +1,6 @@
-{ lib, stdenv, fetchurl, config }:
+{ lib, stdenv, fetchurl, config, dbfile ? config.locate.dbfile or "/var/cache/locatedb" }:
```

Comment:
Tbh I don't know why we have this. Other packages also just hardcode paths.

Reply:
Maybe I just create a PR dropping this entirely if @peterhoeg does not object

Reply:
I guess someone wanted to put their locate database on some external storage?

Reply:
I think this can be accomplished with symlinks which removes some not really required complexity.